exercises/all-your-base/test.sml

```diff
@@ -0,0 +1,55 @@
+(* version 1.1.0 *)
+
+use "all-your-base.sml";
+use "testlib.sml";
+
+infixr |>
+fun x |> f = f x
+
+val testsuite =
+  describe "all-your-base" [
+    test "base_2(1) -> base_10(1)"
```

Comment:
I'll update the descriptions to match canonical-data.json, but yes all of the tests are from there.